test/prober/configuration.go

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2020 The Knative Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package prober
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"path"
+	"runtime"
+	"text/template"
+
+	"github.com/wavesoftware/go-ensure"
```

Comment:
seems like this is an extra dep that is not needed.

Reply:
It's used multiple times through the code with: 
```golang
ensure.NoError(err)
```